src/factories/web3/Web3Modal.ts

```diff
@@ -21,6 +23,14 @@ export default {
     const providerOptions = {
       /* See Provider Options Section */
       ...options,
+
+      // https://infura.io/dashboard/ethereum
+      walletconnect: {
+        package: WalletConnectProvider,
+        options: {
+          infuraId: "ecfafabdcff44308ac6a993e5225b790",
+        },
+      },
```

Comment:
@whatl3y Had to create an Infura account and project for Moontography to get an `infuraId`. If you have already created and account and project, and have an id feel free to replace, otherwise you can create on with an official Moontography email and create an official Moontography project to get an id. See doc below:

https://blog.infura.io/getting-started-with-infura-28e41844cc89/

Reply:
That's alright, can you grant me access to manage this key/ID as well? I'd just like to access if possible as needed.